README.md

```diff
@@ -300,18 +300,21 @@ This currently can only be loaded in a browser and not as an overlay.
 This will work in current version of Chrome,
 and should work in other browsers as well but this is less tested.
 
-Instructions:
+If you want the emulator to use your ACT settings and user triggers, you will need to enable the OverlayPlugin WS Server via the following instructions:
 
 1. Start ACT.
-1. Make sure the WS Server is started via Plugins -> OverlayPlugin WSServer -> Stream/Local Overlay.
-1. Select `Cactbot Raidboss (Combined Alerts and Timelines)` from the URL Generator list.
-1. Edit the url to say `raidemulator.html` instead of `raidboss.html`.
-1. Copy and paste this edited url into Chrome.
+1. Start the WS Server via Plugins -> OverlayPlugin WSServer -> Stream/Local Overlay.
+
+If you're developing triggers for the cactbot repository, you can start a local development server via `npm run start` and load the overlay in Chrome via `http://127.0.0.1:8080/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`
+
+If you're developing user triggers, you can load the overlay in Chrome via `https://quisquous.github.io/cactbot/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws`
```

Comment:
Is it worth adding some text around not being connected to ACT and using the github.io link? (Maybe this would need to be before the "If you want the emulator to use your ACT settings" line, or somewhere else?)

Reply:
I'm not 100% sure what you're asking here, probably because my brain is fried from being outside in 88ºf 85% humidity weather for several hours today?

Reply:
(yikes, re: heat)

To restate it, you list two options `(1) ACT + local dev` and `(2) ACT + github.io`.  I was wondering if it made sense to have a `(3) github.io without ACT` option as well.  Since the raid emulator supports not having ACT connected, it might be an easy way to say "hey could you try out your log in the raid emulator (link to instructions) and see if it repros" without making somebody set up ACT and the ws server.

Reply:
Ah, that makes sense. So in essence:

```markdown
If you're trying to reproduce an issue, you can load the overlay in Chrome via `https://quisquous.github.io/cactbot/ui/raidboss/raidemulator.html`. You don't need the WS Server running in this case.
```

Reply:
Yeah, exactly!

Reply:
Added that comment, also added some additional line breaks per the other conversation where they made sense.